tests/test-utils/makeRequestProcessor.ts

```diff
@@ -42,6 +56,16 @@ export default function makeRequestProcessor(name: string, userOpts?: MakeFuncti
             return next('route');
          } else if (opts.passesErrorToNext) {
             return next(err);
+         } else if (opts.returnsRejectedPromise) {
+            return delay(Promise.reject(`Rejection from "${name}"`), 4);
```

Comment:
If you choose to fix the comment above, you could also give the second parameter a default value of `4` (i.e. `function delay(value: unknown, ms: number = 4): Promise<unknown> {`) so you don't have to keep passing `4` every time you use the `delay` function. Not a big deal though... just seemed better.